tel2puml/otel_to_pv/sequence_otel_v2.py

```diff
@@ -357,7 +357,7 @@ def otel_to_pv(
     async_flag: bool = False,
     event_to_async_group_map: dict[str, dict[str, str]] | None = None,
 ) -> Generator[
-    tuple[str, Generator[Generator[PVEvent, Any, None], Any, None]], Any, None
+    tuple[str, Generator[Iterable[PVEvent], Any, None]], Any, None
```

Comment:
Is this passing mypy?

Reply:
A Generator is an Iterable but Iterable's aren't always Generators i.e. Generator is more focussed than an Iterable. We are actually returning a Generator if I'm not mistaken so I think we should probably return it to the previous